doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -614,148 +619,101 @@ syntax: :n:`let fix @ident @binders := @term in` stands for
-             {? Local } Axioms {+ ( {+ @ident } : @type ) }
-             {? Local } Conjecture {+ ( {+ @ident } : @type ) }
-             {? Local } Conjectures {+ ( {+ @ident } : @type ) }
-      :name: Parameters; Axiom; Axioms; Conjecture; Conjectures
+   This command binds one or more :n:`@ident`\s to specified :n:`@type`\(s) as their specifications in
+   the global context. The fact asserted by the :n:`@type` is accepted as a
+   postulate.
 
-      These variants are synonyms of :n:`{? Local } Parameter {+ ( {+ @ident } : @type ) }`.
+   :cmd:`Axiom`, :cmd:`Conjecture`, :cmd:`Parameter` and their plural forms
+   are equivalent.  They can take the :attr:`local` attribute (see :ref:`gallina-attributes`),
+   which makes the defined :n:`@ident`\s accessible by :cmd:`Import` and its variants
+   only through their fully qualified names.
 
-   .. cmdv:: Variable  {+ ( {+ @ident } : @type ) }
-             Variables {+ ( {+ @ident } : @type ) }
-             Hypothesis {+ ( {+ @ident } : @type ) }
-             Hypotheses {+ ( {+ @ident } : @type ) }
-      :name: Variable (outside a section); Variables (outside a section); Hypothesis (outside a section); Hypotheses (outside a section)
+   Similarly, :cmd:`Hypothesis`, :cmd:`Variable` and their plural forms are equivalent.  Outside
+   of a section, these are equivalent to :n:`Local Parameter`.  Inside a section, the
+   :n:`@ident`\s defined are only accessible within the section.  When the current section
+   is closed, the :n:`@ident`\s
+   becomes undefined and every object using this variable will be explicitly
+   parameterized (i.e., the variable is *discharged*).  See Section :ref:`section-mechanism`.
 
-      Outside of any section, these variants are synonyms of
-      :n:`Local Parameter {+ ( {+ @ident } : @type ) }`.
-      For their meaning inside a section, see :cmd:`Variable` in
-      :ref:`section-mechanism`.
+   .. exn:: @ident already exists.
+      :name: @ident already exists. (Axiom)
+      :undocumented:
 
-      .. warn:: @ident is declared as a local axiom [local-declaration,scope]
+   .. warn:: @ident is declared as a local axiom [local-declaration,scope]
 
-         Warning generated when using :cmd:`Variable` instead of
-         :cmd:`Local Parameter`.
+      Warning generated when using :cmd:`Variable` instead of
+      :n:`Local Parameter`.
 
 .. note::
-   It is advised to use the commands :cmd:`Axiom`, :cmd:`Conjecture` and
+   We advise using the commands :cmd:`Axiom`, :cmd:`Conjecture` and
    :cmd:`Hypothesis` (and their plural forms) for logical postulates (i.e. when
-   the assertion :token:`type` is of sort :g:`Prop`), and to use the commands
+   the assertion :n:`@type` is of sort :g:`Prop`), and to use the commands
    :cmd:`Parameter` and :cmd:`Variable` (and their plural forms) in other cases
    (corresponding to the declaration of an abstract mathematical entity).
 
-.. seealso:: Section :ref:`section-mechanism`.
-
```

Comment:
Here is an example that could be added at this point (to show some variants of the syntax):

```coq
Parameter X Y : Set.
Parameter (R : X -> Y -> Prop) (S : Y -> X -> Prop).
Axiom R_S_inv : forall x y, R x y <-> S y x.
```